docs/howto/openstack/nova/server-group.md

```diff
@@ -0,0 +1,84 @@
+
+### Soft affinity
+
+A policy of `soft-affinity` will **try to** make sure that all the servers in that group are placed on the same physical compute node, but ultimately will allow it if otherwise not possible.
+
+### Anti-affinity
+
+A server group with the policy of `anti-affinity` will make sure that the servers in that group are **never** placed on the same physical compute node.
+
+### Soft anti-affinity
+
+A policy of `soft-anti-affinity` will **try to** make sure that all the servers in that group are not placed on the same physical compute node, but ultimately will allow it if otherwise not possible.
+
+## Creating server groups
+
+To create a server group, use the following command:
+
+```bash
+openstack server group create \
+  --policy [affinity|soft-affinity|anti-affinity|soft-anti-affinity] \
+  <server_group_name>
+```
+
+With an `anti-affinity` or `soft-anti-affinity` policy, you may also configure how many servers you want to allow on the same physical compute node. To do this, use the option `--rule max_server_per_host=<number>`, where `<number>` is the amount of servers to allow on the same physical compute node.
+
+
+## Creating servers using server groups
+
+To apply a server group policy, you must specify the group when creating a server, as a *scheduling hint.* To do that, use the `--hint` parameter in the following command:
+
+```bash
+openstack server create --hint group=<server_group_id> [...] <server_name>
+```
+
+If you subsequently launch more servers referencing the same server group, {{brand}} concentrates or distributes them according to the server group's policy.
+
+## Troubleshooting common issues
+
+If you keep creating servers within a server group with a policy of `anti-affinity`, you will eventually exceed the total amount of physical compute nodes in the region. The command will still succeed but the server will fail in scheduling, with the following "fault" message: _No valid host was found. There are not enough hosts available._
+
+```console
+$ openstack server show -c fault -c status <server_id>
++--------+--------------------------------------------------+
+| Field  | Value                                            |
++--------+--------------------------------------------------+
+| fault  | {'code': 500, 'created': '2022-12-23T11:21:33Z', |
+|        | 'message': 'No valid host was found.             |
+|        | There are not enough hosts available.'}          |
+| status | ERROR                                            |
++--------+--------------------------------------------------+
+```
+
+This is normal as {{brand}} cannot schedule the server on a different physical compute node because there already is a server in the server group on every node. The same scheduling error and "fault" message will occur when using a server group with a policy of `affinity` as well, when you create more servers than a physical compute node can host.
```

Comment:
Instead of "because there already is a server in the server group" I would say "because there is a server in the server group already".

Reply:
Nope. :)